.github/workflows/eth-bytecode-db-extractors.yml

```diff
@@ -2,6 +2,9 @@ on:
   push:
     branches:
       - 'main'
+    paths:
+      - eth-bytecode-db-extractors/**
+      - .github/workflows/eth-bytecode-db-extractors.yml
```

Comment:
maybe add `      - .github/actions/deps/**` ?

Reply:
The extractors do not use deps